src/plugins/console/public/application/containers/settings.tsx

```diff
@@ -8,6 +8,7 @@
 
 import React from 'react';
 
+import { HttpSetup } from 'kibana/public';
```

Comment:
nit: recommendation for future if you need to import only type please use the following format: 
`import type { HttpSetup } from 'kibana/public';`